chrome/locale/en-US/zotero/zotero.ftl

```diff
@@ -474,6 +474,8 @@ quickformat-aria-item = Press { return-or-enter } to select this item. Press Tab
 quickformat-accept = 
     .tooltiptext = Save edits to this citation
 
+insert-note-aria-input = Type to search through your notes. Press Tab to begin navigating the results. Press Escape to close the dialog.
+insert-note-aria-item = Press Enter to select this note. Use Arrow Up, Arrow Down to navigate search results. Press Tab to go back to the search field. Press Escape to close the dialog.
```

Comment:
These should match the QF strings above in guidance and terminology.

Reply:
QF strings are being revised in #4004 to hopefully make QF instructions more descriptive. So I guess it's best to wait for that be be finalized first